osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -87,6 +89,13 @@ public DrawableSlider(Slider s)
             }
         }
 
+        [BackgroundDependencyLoader]
+        private void load(OsuConfigManager config)

```

Comment:
why was this moved here?

Reply:
Because it shouldn't be done at such a low level like `SliderBody` - something that is exposed publicly for the purpose of drawing a slider body. We may want to use `SliderBody` without having an entire hit object.